log/log.go

```diff
@@ -123,6 +125,36 @@ func Error(ctx context.Context, fields map[string]interface{}, format string, ar
 			if err == nil {
 				entry = entry.WithField("identity_id", identityID)
 			}
+
+			if req := goa.ContextRequest(ctx); req != nil {
```

Comment:
But.. hmm.. this will add that to each Error line. I guess we need to control in general what is logged as error. But wouldn't it be better to log this with the 'Request complete' log when e.g. status != 2xx ?